Framework/PythonInterface/mantid/CMakeLists.txt

```diff
@@ -35,35 +35,36 @@ add_custom_target ( PythonModule DEPENDS ${PYTHON_PKGROOT_INSTALL_FILES} ${PYTHO
 set_property ( TARGET PythonModule PROPERTY FOLDER "MantidFramework/Python" )
 
 ####################################################################################
-# Sub modules 
+# Sub modules
 ####################################################################################
 add_subdirectory ( kernel )
 add_dependencies ( PythonKernelModule PythonModule ) # Ensure the module files are in place
 add_subdirectory ( geometry )
 add_subdirectory ( api )
 add_subdirectory ( dataobjects )
 add_subdirectory ( _plugins )
+add_subdirectory ( plots )
 
 # Create an overall target
-add_custom_target ( PythonInterface DEPENDS PythonKernelModule PythonGeometryModule 
-                    PythonAPIModule PythonDataObjectsModule PythonCurveFittingModule )
+add_custom_target ( PythonInterface DEPENDS PythonKernelModule PythonGeometryModule
+                    PythonAPIModule PythonDataObjectsModule PythonCurveFittingModule
+                    PythonPlotsModule)
```

Comment:
Can you change the framework (CMakeLists.txt)[/mantidproject/mantid/blob/master/Framework/CMakeLists.txt#L147] and remove the `Python*` targets and replace it with `PythonInterface`. This should ensure a `make Framework` gets everything.